extensions/schema-compare/src/models/schemaCompareOptionsModel.ts

```diff
@@ -9,89 +9,218 @@ import { isNullOrUndefined } from 'util';
 export class SchemaCompareOptionsModel {
 	public deploymentOptions: mssql.DeploymentOptions;
 	public excludedObjectTypes: mssql.SchemaObjectType[] = [];
+	public optionsMapTable: Record<string, mssql.DacDeployOptionPropertyBoolean> = {};
```

Comment:
any reason for using Record instead of Map?

Reply:
No specific reason, just used intially type instead of map directly and never got a chance to look into it, have updated with Map ds. Thanks